index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
```

Comment:
- [ ] I think it would be better if you wrap your title/logo in an anchor `<a>` tag instead of a div `so that it offers a good user experience to your users as it will be easier for them to navigate the website 😄 and it will also make your site semantic 👍🏼 .